recipes-tag/04-id-isr-collection/meta.yaml

```diff
@@ -11,9 +11,7 @@ build:
 
 requirements:
   run:
-    - gsl
-    - libiconv
-    - pcre
     - hkl
```

Comment:
I don't think we need hkl explicitly here. It's taking care of by hklpy itself: https://github.com/NSLS-II/lightsource2-recipes/blob/5e3f1850af2a45ba011e1b554d4318510871beb3/recipes-tag/hklpy/meta.yaml#L21